charts/README.md

```diff
@@ -79,6 +79,10 @@ Parameter | Description | Default
 `router.roundTrip.timeout` | HTTP transport request timeout | `50ms`
 `router.roundTrip.timeoutExponent` | The length of request timeout will multiply with timeoutExponent after each retry | `2` 
 `router.roundTrip.maxRetries` | Max retries times of a failed request | `10`
+`openTracing.enabled` | If true, OpenTracing is enabled | `false`
+`openTracing.collectorEndpoint` | Jaeger collector endpoint | ``
+`openTracing.samplingRate` | Probabilistic sampling rate | `0.5`
+`otelCollectorEndpoint` | OpenTelemetry collector endpoint | None
```

Comment:
Should we separate section for `openTelemetry` ?

```
openTelemetry:
         enabled: false
         collectorEndpoint: xyz
```

OR we can have `tracingMode` which can be either of `opentracing` , `opentelemetry` or `none`
then specific section for each mode.